firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionEvents.kt

```diff
@@ -74,7 +75,7 @@ internal object SessionEvents {
       androidAppInfo =
         AndroidApplicationInfo(
           packageName = packageName,
-          versionName = packageInfo.versionName,
+          versionName = packageInfo.versionName ?: buildVersion,
```

Comment:
Is build version guaranteed?

Reply:
The `buildVersion` is the `versionCode` as a String. When no versionCode is set, it returns 0. Play Store won't let apps upload two versions with the same versionCode. So it'll never be null, and it'll always be unique for apps published to the Play Store.